src/snowflake/cli/api/secure_path.py

```diff
@@ -271,20 +271,20 @@ def temporary_directory(cls):
             log.info("Removing temporary directory %s", tmpdir)
 
     def _assert_exists_and_is_file(self) -> None:
-        self._assert_exists()
-        self._assert_is_file()
```

Comment:
Separate to other change to as this change seems to be unrelated.

Reply:
Done: https://github.com/snowflakedb/snowflake-cli/pull/875